coyote/views.py

```diff
@@ -24,6 +24,11 @@ def get_domain(url):
 
 
 def home(request):
+    """
+    it takes the inputted code from frontend request and sends it to hackerearth API
```

Comment:
Just add `Parameters` and `Returns` too. Refer #11 for the format. Rest all good.
Nice job @142ayushkumar!